db/migrate/20160324114924_add_index_to_create_subjects.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexToCreateSubjects < ActiveRecord::Migration
+  def change
+    add_column :create_subjects, :name, :string
+    add_index :create_subjects, :name, unique: true
```

Comment:
yeah i realized i had the table name wrong here, but what do you mean by column name?